galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
They probably shouldn't, that would seem to be a bug. I'll look into that tomorrow.